packages/docs/src/components/HeaderBanner.vue

```diff
@@ -23,13 +23,26 @@
         </a>
         <a href="https://nuxtjs.org/partners/epicmax" target="_blank" class="link-5"><strong class="bold-text-9">Nuxt</strong></a>
       </div>
-      <div class="header__banner-button">
+      <div class="header-banner__button">
         <va-button href="https://epicmax.co/contacts?ref=vuestic-header" target="blank">{{ $t("landing.header.banner.hire") }}</va-button>
+        <div class="header-banner__button--close">
+          <va-icon class="cursor-pointer" size="small" name="close" @click="hide" />
+        </div>
       </div>
     </div>
   </div>
 </template>
 
+<script setup lang="ts">
+import { ref } from 'vue'
+const isHidden = ref(localStorage.getItem('official-partner-banner') === 'hide')
```

Comment:
```suggestion
const isHidden = ref(localStorage.getItem('banner') === 'official-partner')
```

The idea is that only one banner should be available, and we also do not want to keep a key in local storage for each banner historically.

Reply:
got it